bgpd/bgp_nexthop.c

```diff
@@ -240,6 +240,40 @@ bgp_addr_onlink_v4 (struct in_addr *addr)
   return 0;
 }
 
+/* If the address exists on connected networks return 1. */
+int
+bgp_addr_onlink (afi_t afi, union sockunion *su)
```

Comment:
Probably you need delete bgp_addr_onlink_v4()

Reply:
done